src/hazelcore/CursorInfo_Exp.re

```diff
@@ -868,18 +892,19 @@ and ana_cursor_info_zoperand =
           ),
         )
       };
+    | Case(InconsistentBranches(_, _, Ana), _, _) =>
```

Comment:
this is an impossible case I think? Ana should only appear if the case is actually in analytic position?